reference-implementation/to-upstream-wpts/resources/recording-streams.js

```diff
@@ -0,0 +1,74 @@
+"use strict";
+
+self.recordingReadableStream = (extras = {}, strategy) => {
+  let controllerToCopyOver;
+  const stream = new ReadableStream({
+    start(controller) {
+      controllerToCopyOver = controller;
+
+      if (extras.start) {
```

Comment:
Oh gross. Definitely not in this repo; I think we have a lint rule disallowing such elision :)